default.nix

```diff
@@ -0,0 +1,34 @@
+{ # Fetch the latest haskell.nix and import its default.nix
+  haskellNix ? import (builtins.fetchTarball "https://github.com/input-output-hk/haskell.nix/archive/master.tar.gz") {}
```

Comment:
Let use [Niv](https://github.com/nmattia/niv) and pin a specific revision.

Reply:
will do!